iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractTapChangerAdderImpl.java

```diff
@@ -95,12 +101,18 @@ public T add() {
                 network.setValidationLevelIfGreaterThan(ValidationLevel.EQUIPMENT);
             }
         }
+        if (solvedTapPosition != null) {
+            int highTapPosition = lowTapPosition + steps.size() - 1;
+            if (solvedTapPosition < lowTapPosition || solvedTapPosition > highTapPosition) {
+                LOGGER.warn("{} has incorrect solved tap position {}. It is not within bounds [{}, {}].", parent, solvedTapPosition, lowTapPosition, highTapPosition);
```

Comment:
I am not sure about this behaviour: 
if the solved tap position is wrong when we create the tap changer, what should happen? The validation level of the network should stay the same but do we want to allow the creation of the solved tap position? And to what should it be set? null?